tx/textual/valuerenderer/types.go

```diff
@@ -2,11 +2,17 @@ package valuerenderer
 
 import (
 	"context"
-	"io"
 
 	"google.golang.org/protobuf/reflect/protoreflect"
 )
 
+// Item is the abstract unit of Textual rendering.
+type Item struct {
```

Comment:
talked offline with Jim, let's rename to Screen

Reply:
Done.